mobify/platform-scaffold
# WEBDATA-154 Multiple coupon codes

This PR introduce the ability to handle multiple coupon codes.

**Note: Magento does not support multiple coupon codes at the moment, try it on SFCC connector! Hybris support will be in a separate PR.**

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-154
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/850

## Changes
- SFCC promo commands: `putPromoCode`, `deletePromoCode`
- display crossed out original price when price is reduced by coupon
- Update SFCC basket parser
- Cart summary and order summary style changes

## Todos

## How to test-drive this PR
- Make sure you are using SFCC connector, `npm link` with SDK https://github.com/mobify/progressive-web-sdk/pull/850
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add product to cart (order total needs to be more than $100, `AMAZE` coupon code rule)
- Apply multiple coupons: `MAGIC`, `AMAZE`
- Verify you can add and delete promo codes in cart summary and order summery
